test/integration/test_utils.mjs

```diff
@@ -34,9 +37,8 @@ function loadAndWait(filename, selector, zoom, pageSetup) {
       });
 
       let url = `${global.integrationBaseUrl}?file=/test/pdfs/${filename}`;
-      if (zoom) {
-        url += `#zoom=${zoom}`;
-      }
+      url += `#zoom=${zoom ?? "page-fit"}`;
```

Comment:
This feels somewhat unrelated to the title "Fix integration tests on Mac", why do we need this?

Reply:
I've a macbook and its screen is not that big, so some element we want to click on weren't visible.
So instead of adding a lot of scrollIntoView everywhere it was easier to just show all the page, so it's related.